aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsCrateDocsDecorator.kt

```diff
@@ -36,96 +48,176 @@ class AwsReadmeDecorator : ClientCodegenDecorator {
 
-                    ```rust
-                    use $snakeCaseModuleName as $shortModuleName;
+            The SDK provides one crate per AWS service. You must add [Tokio](https://crates.io/crates/tokio)
+            as a dependency within your Rust project to execute asynchronous code. To add `$moduleName` to
+            your project, add the following to your **Cargo.toml** file:
 
-                    #[tokio::main]
-                    async fn main() -> Result<(), $shortModuleName::Error> {
-                        let config = aws_config::load_from_env().await;
-                        let client = $shortModuleName::Client::new(&config);
+            ```toml
+            [dependencies]
+            aws-config = "$awsConfigVersion"
+            $moduleName = "${codegenContext.settings.moduleVersion}"
+            tokio = { version = "1", features = ["full"] }
+            ```
 
-                        // ... make some calls with the client
+            Then in code, a client can be created with the following:
 
-                        Ok(())
-                    }
-                    ```
+            ```rust,$exampleMode
+            use $snakeCaseModuleName as $shortModuleName;
 
-                    See the [client documentation](https://docs.rs/$moduleName/latest/$snakeCaseModuleName/client/struct.Client.html)
-                    for information on what calls can be made, and the inputs and outputs for each of those calls.
+            ##[#{tokio}::main]
+            async fn main() -> Result<(), $shortModuleName::Error> {
+                let config = #{aws_config}::load_from_env().await;
+                let client = $shortModuleName::Client::new(&config);
 
-                    ## Using the SDK
+                // ... make some calls with the client
 
-                    Until the SDK is released, we will be adding information about using the SDK to the
-                    [Developer Guide](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/welcome.html). Feel free to suggest
-                    additional sections for the guide by opening an issue and describing what you are trying to do.
+                Ok(())
+            }
+            ```
+
+            See the [client documentation](https://docs.rs/$moduleName/latest/$snakeCaseModuleName/client/struct.Client.html)
+            for information on what calls can be made, and the inputs and outputs for each of those calls.${"\n"}
+            """.trimIndent().trimStart(),
+            "tokio" to CargoDependency.Tokio.toDevDependency().toType(),
+            "aws_config" to when (compileExample) {
+                true -> AwsCargoDependency.awsConfig(codegenContext.runtimeConfig).toDevDependency().toType()
+                else -> writable { rust("aws_config") }
+            },
+        )
 
-                    ## Getting Help
+        template(
+            asComments,
+            """
+            #### Using the SDK
```

Comment:
_web dev semantic HTML nitpick_: We should be using `h2` elements for these headings instead of `h3`.
```suggestion
            ### Using the SDK
```

Reply:
This is actually `##` after escaping is taken into account.